metadata-ingestion/src/datahub/ingestion/source/dataplex/dataplex_config.py

```diff
@@ -0,0 +1,202 @@
+    )
+
+    extract_zones: bool = Field(
+        default=True,
+        description="Whether to extract Zone metadata as Containers.",
+    )
+
+    extract_assets: bool = Field(
+        default=True,
+        description="Whether to extract Asset metadata as Containers.",
+    )
+
+    extract_entities: bool = Field(
+        default=True,
+        description="Whether to extract Entity metadata (discovered tables/filesets) as Datasets.",
+    )
+
+    extract_entry_groups: bool = Field(
+        default=False,
+        description="Whether to extract Entry Groups from Universal Catalog. (Phase 2 feature)",
+    )
+
+    extract_entries: bool = Field(
+        default=False,
+        description="Whether to extract Entries from Universal Catalog. (Phase 2 feature)",
+    )
+
+    extract_lineage: bool = Field(
+        default=True,
+        description="Whether to extract lineage information using Dataplex Lineage API. "
+        "Extracts table-level lineage relationships between entities.",
+    )
+
+    lineage_fail_fast: bool = Field(
+        default=False,
+        description="If True, fail immediately on lineage extraction errors. "
+        "If False (default), log errors and continue processing other entities. "
+        "Recommended: False for development, True for production to catch issues early.",
+    )
+
+    max_lineage_failures: int = Field(
+        default=10,
+        description="Maximum number of consecutive lineage extraction failures before stopping. "
+        "Acts as a circuit breaker to prevent excessive API calls when there's a systemic issue. "
+        "Set to -1 to disable circuit breaker. Default: 10.",
+    )
+
+    lineage_batch_size: int = Field(
+        default=1000,
+        description="Number of entities to process in each lineage extraction batch. "
+        "Lower values reduce memory usage but may increase processing time. "
+        "Set to -1 to disable batching (process all entities at once). "
+        "Recommended: 1000 for large deployments, -1 for small deployments. Default: 1000.",
+    )
+
+    create_sibling_relationships: bool = Field(
+        default=True,
+        description="Whether to create sibling relationships between Dataplex entities and native platform datasets (BigQuery/GCS).",
+    )
+
+    dataplex_is_primary_sibling: bool = Field(
+        default=False,
+        description="Controls sibling relationship primary designation between Dataplex entities and source platform entities. "
+        "When False (default), source platform entities (BigQuery/GCS) are primary and Dataplex entities are secondary. "
+        "When True, Dataplex entities are primary and source platform entities are secondary. "
+        "Default is False since source platforms are typically the canonical data source.",
+    )
+
+    apply_zone_type_tags: bool = Field(
+        default=True,
+        description="Whether to apply tags based on zone types (RAW, CURATED).",
+    )
```

Comment:
so Zone can become Container and/or Tag, right?

Reply:
This is confusing, made these zones a custom property instead of a tag as the default behavior and avoided the config for simplicity.